src/librand/distributions/ziggurat_tables.rs

```diff
@@ -12,8 +12,8 @@
 // algorithm. Autogenerated by `ziggurat_tables.py`.
 
 pub type ZigTable = &'static [f64; 257];
-pub static ZIG_NORM_R: f64 = 3.654152885361008796;
-pub static ZIG_NORM_X: [f64; 257] =
```

Comment:
I feel like these are good examples of `static`s instead of constants, as there's no need to inline this value anywhere and it's large enough to deserve it's own slot in memory.